setup.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 from setuptools import setup, find_packages
+import opmd_viewer  # In order to extract the version number
```

Comment:
this is part I am not 100% sure in the original blog post with:

the include has quite some dependencies. we could also create a single version.py file that we both include in `setup.py` and in ` o[pen]pmd_viewer/__init__.py`. By that we could depend on a more lightweight include with fewer side effects and still have the version number in one place for both the packaging and the module.